metric/registry.go

```diff
@@ -61,13 +62,21 @@ func WithUnit(unit metricdata.Unit) Options {
 	}
 }
 
-// WithLabelKeys applies provided label.
+// WithLabelKeys applies provided label keys.
 func WithLabelKeys(labelKeys ...string) Options {
 	return func(mo *metricOptions) {
 		mo.labelkeys = labelKeys
 	}
 }
 
+// WithConstLabel applies provided constant label.
+func WithConstLabel(labelKey, labelValue string) Options {
```

Comment:
it might be better to take multiple pairs of constant key/value. Perhaps a map would be better.
```go
func WithConstLabel(constLabel map[LabelKey]LabelValue) Options {
}

Reply:
changed.